test/integration/Cardano/Wallet/Network/HttpBridgeSpec.hs

```diff
@@ -140,6 +146,10 @@ spec = do
             (show err) `shouldContain`
                 "Transaction failed verification: transaction has more witnesses than inputs"
   where
+    actionDummy :: [Block] -> IO ()
+    actionDummy b = do
+      putStr (show b)
```

Comment:
There were so many possibilities. So many things you _could_ do with this `Block`. And you choose to print it!

I am so disappointed :trollface: